stories/styleguide/lws.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+
+import { LWSModalBody } from '../../src/lws/lws-modal-body';
+import { LWSModalHeader } from '../../src/lws/lws-modal-header';
+import { LWSSelectWallet } from '../../src/lws/lws-select-wallet';
+import { LWSRequiredInfo } from '../../src/lws/lws-required-info';
+import { LWSWalletConnectionError } from '../../src/lws/lws-wallet-connection-error';
+import { LWSSelfkeyIdError } from '../../src/lws/lws-selfkey-id-error';
+import { LWSExtensionError } from '../../src/lws/lws-extension-error';
+import { LWSSuccess } from '../../src/lws/lws-success';
+
+
+
+
+
+export const LWSSelectWalletModal = () => (
+  <div>
+    <LWSModalHeader/>
+    <LWSModalBody>
+      <LWSSelectWallet wallets={[{publicKey: '0x4184288c556524df9cb9e58b73265ee66dca4efe'}]}/>
```

Comment:
do we need to make data dynamic and some kind of standardized tags for working in UI?

Reply:
@altninja here wallets are going to come from the redux so with redux we can map the data in the store to props that the component is expecting. In this case we will need to get the data that comes from Identity wallet and pick a list of wallet public keys for this component. Does it make sense? Cheers